.github/workflows/main.yml

```diff
@@ -27,3 +26,18 @@ jobs:
             echo $CHANGES
             exit 1
           fi
+
+  build:
```

Comment:
Translation check was setup for `workflow_dispatch` where you can manually trigger a run of it in github actions as needed. Do we think adding this here will cause any issues with that?

Reply:
Thanks for pointing it out. I separated them.